File: app/src/main/java/com/ddancn/keepaccount/constant/TypeEnum.java
package com.ddancn.keepaccount.constant;

/**
 * @author ddan.zhuang
 * @date 2019/10/16
 */
public enum TypeEnum {
    /**
     * 收入
     */
    IN(1),
    /**
     * 支出
     */
    OUT(-1);

    private int value;

    TypeEnum(int value){
        this.value = value;
    }

    public int value() {
        return value;
    }
}

File: app/src/main/java/com/ddancn/keepaccount/constant/TypeEnum.kt
package com.ddancn.keepaccount.constant

/**
 * @author ddan.zhuang
 * @date 2019/10/16
 */
enum class TypeEnum constructor(private val value: Int) {
    /**
     * 收入
     */
    IN(1),
    /**
     * 支出
     */
    OUT(-1);

    fun value(): Int {
        return value
    }
}
